index.html

```diff
@@ -254,7 +259,7 @@ <h3>Resources Included in the <a>PerformanceResourceTiming</a>
 <a data-cite=
 "PERFORMANCE-TIMELINE-2#performance-timeline">Performance
 Timeline</a>, unless excluded from the timeline as part of the <a
-href="#processing-model">processing model</a>.
+href="FETCH#concept-fetch">fetching process</a>.
```

Comment:
This link does not seem to work in the Preview. Is this a Preview bug or does it need to be fixed?

Reply:
fixed